### PR TITLE
v1.18.0r1

### PR DESCRIPTION
**Pull request by shefty**

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -227,22 +234,23 @@ Ubuntu <ubuntu@ip-172-31-15-224.ec2.internal>
 Venkata Krishna Nimmagadda <nvkrishna85@gmail.com>
 Venkata Krishna Nimmagadda <venkata.krishna.nimmagadda@intel.com>
 Vishwas Dsouza <vidsouza@amazon.com>
-Wei Zhang <wzam@amazonc.com>
 Wei Zhang <wzam@amazon.com>
+Wei Zhang <wzam@amazonc.com>
```
Comment (shijin-aws) [inline AUTHORS:238]: is this a typo?

Comment (shefty) [reply AUTHORS:238]: This was updated based on a script, so the typo likely exists in the commit itself.  I don't know why the script resulted in so many updates to this file, as it's the same script that I alway use.

Comment (shijin-aws) [reply AUTHORS:238]: I see, nvm, thanks!

**Comment by shefty** [comment]
memo to me: disable sm2 from build

File: NEWS.md [hunk 2]
```diff
@@ -6,52 +6,245 @@ bug fixes (and other actions) for each version of Libfabric since
...
+========================
+
+## Core
+
+- Fix spinlocks for macOS
+- hmem_cuda Add const to param to remove warning
+- Fix typos in fi_ext.h
+- ofi_epoll: Remove unused hot_index struct member
 
 ## EFA
```
Comment (lrbison) [inline NEWS.md:49]: Why are there two EFA sections? Other is on line 123

Comment (shefty) [reply NEWS.md:49]: These updates are copied from the readme file in the v1.17.x branch.  One contains the updates for the v1.17.0 release, the other v1.17.1.  The updates for the v1.18.0 release is empty and at the top of the file.

**Comment by shefty** [comment]
@shijin-aws @wckzhang - I need to create rc1.  I noticed there are several EFA PRs open marked for-v1.18.  Are those close to ready or okay to merge after I create rc1 and can create the v1.18.x branch?

**Comment by wzamazon** [comment]
> I noticed there are several EFA PRs open marked for-v1.18. Are those close to ready or okay to merge?

These PRs marked as 1.18.x are close to be merged. I think we will merge them today. 

I am working on another PR, and will be open today.

> after I create rc1 and can create the v1.18.x branch?

I am not sure I understand. Do you mean you will create 1.18.x now?

Can we back port PRs currently open?

**Comment by shefty** [comment]
Once the v1.18.x branch is created, there's no problem backporting updates to it.  But at some point, I need to create that branch and rc1.  I can do that as soon as #8669 CI completes (again...).  Or, if you'll be ready to merge the open PRs today, I can wait for those, integrate them, and then create rc1.  Creating the package is trivial -- the delay is needing to re-run CI after integrating new upates.

**Comment by wzamazon** [comment]
I do not think you need to wait. We can backport them to 1.18.x branch

**Pull request merged**